backend/app/views/spree/admin/users/_form.html.erb

```diff
@@ -8,15 +8,14 @@
 
     <div data-hook="admin_user_form_roles" class="form-group">
       <strong><%= Spree.t(:roles) %></strong>
-      <% @roles.each do |role| %>
+      <%= f.collection_check_boxes :spree_role_ids, Spree::Role.all, :id, :name do |role| %>
         <div class="checkbox">
-          <%= label_tag "user_spree_role_#{role.name}" do %>
-            <%= check_box_tag 'user[spree_role_ids][]', role.id, @user.spree_roles.include?(role), :id => "user_spree_role_#{role.name}" %>
-            <%= role.name %>
+          <%= role.label do %>
+            <%= role.check_box %>
```

Comment:
@Bharat311 - I guess we should rename role here to role_form or something like that, so it will remove the confusion that it is not a role but its form.

any thoughts?